FESS-GUI/windowsnames.h

```diff
@@ -2,6 +2,12 @@
 #define WINDOWSNAMES
 
 #define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
+#define MAINWINDOW_ERROR "FlyCAM - Error"
```

Comment:
![screenshot 4](https://cloud.githubusercontent.com/assets/5752455/20235024/2c39d12e-a83c-11e6-80ef-140bcf672bbb.png)
This is from my PR